### PR TITLE
DOC-3367: Added v22.1.0-beta.4 release note

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-3367

### CODE REVIEW

**Review by lnhsingh — COMMENTED** [review]
A couple nits. Once fixed, LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13699#-:-N0WpIi_DNRS2Mjikodv:bdo8det)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @gemma-shay and @nickvigilante)

___
*[`_includes/releases/v22.1/v22.1.0-beta.4.md` line 9 at r1](https://reviewable.io/reviews/cockroachdb/docs/13699#-N0Wp3dm527nR38rqkC_:-N0Wp3dm527nR38rqkCa:b-ugxmuz) ([raw file](https://github.com/cockroachdb/docs/blob/1a1b328bc961394abcda0f9086a4716a8a4faccf/_includes/releases/v22.1/v22.1.0-beta.4.md#L9)):*
> ```Markdown
> <h3 id="v22-1-0-beta-4-backward-incompatible-changes">Backward-incompatible changes</h3>
> 
> - A user can no longer define the subdirectory of their full backup. This deprecated syntax can be enabled by changing the new `bulkio.backup.deprecated_full_backup_with_subdir` cluster setting to true. [#80145][#80145]
> ```

Link to backup page and cluster setting page
___
*[`_includes/releases/v22.1/v22.1.0-beta.4.md` line 45 at r1](https://reviewable.io/reviews/cockroachdb/docs/13699#-N0WoQLT424jKgEEuMsf:-N0WoQLT424jKgEEuMsg:bui8lkv) ([raw file](https://github.com/cockroachdb/docs/blob/1a1b328bc961394abcda0f9086a4716a8a4faccf/_includes/releases/v22.1/v22.1.0-beta.4.md#L45)):*
> ```Markdown
> <h4 id="v22-1-0-beta-4-backup-change">Backup change</h4>
> 
> - The `sql.multi_region.allow_abstractions_for_secondary_tenants.enabled` [cluster setting](../v22.1/cluster-settings.html) is now consulted when a secondary tenant tries to restore a multi-region database. [#80013][#80013]
> ```

I don't think this should be included in release notes since it's about a cluster setting that has to do with tenants. If we do include it, I would move it to SQL changes (?)

Either way, I don't think it should be under the Miscellaneous > Backup change headings

**Review by nickvigilante — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13699)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @gemma-shay and @lnhsingh)

___
*[`_includes/releases/v22.1/v22.1.0-beta.4.md` line 9 at r1](https://reviewable.io/reviews/cockroachdb/docs/13699#-N0Wp3dm527nR38rqkC_:-N0WrBfXBM3QYvKLtUIX:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/1a1b328bc961394abcda0f9086a4716a8a4faccf/_includes/releases/v22.1/v22.1.0-beta.4.md#L9)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

Link to backup page and cluster setting page
</blockquote></details>

Done.
___
*[`_includes/releases/v22.1/v22.1.0-beta.4.md` line 45 at r1](https://reviewable.io/reviews/cockroachdb/docs/13699#-N0WoQLT424jKgEEuMsf:-N0WrC2K0iSatM09jGaR:b-d45pw7) ([raw file](https://github.com/cockroachdb/docs/blob/1a1b328bc961394abcda0f9086a4716a8a4faccf/_includes/releases/v22.1/v22.1.0-beta.4.md#L45)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

I don't think this should be included in release notes since it's about a cluster setting that has to do with tenants. If we do include it, I would move it to SQL changes (?)

Either way, I don't think it should be under the Miscellaneous > Backup change headings
</blockquote></details>

Omitted.

File: _includes/releases/v22.1/v22.1.0-beta.4.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+## v22.1.0-beta.4
+
+Release Date: April 26, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-4-backward-incompatible-changes">Backward-incompatible changes</h3>
+
+- A user can no longer define the subdirectory of their full backup. This deprecated syntax can be enabled by changing the new `bulkio.backup.deprecated_full_backup_with_subdir` [cluster setting](../v22.1/cluster-settings.html) to true. [#80145][#80145]
```
Comment (gemma-shay) [inline _includes/releases/v22.1/v22.1.0-beta.4.md:9]:
```suggestion
- Users can no longer define the subdirectory of their full backup. This deprecated syntax can be enabled by changing the new `bulkio.backup.deprecated_full_backup_with_subdir` [cluster setting](../v22.1/cluster-settings.html) to `true`. [#80145][#80145]
```

**Review by gemma-shay — APPROVED** [review]
one nit but lgtm!

**Pull request merged**